test/remote/gateways/remote_credorax_test.rb

```diff
@@ -397,12 +397,12 @@ def test_failed_referral_cft
   end
 
   def test_successful_credit
-    response = @gateway.credit(@amount, @credit_card, @options)
+    response = @gateway.credit(@amount, @credit_card, @options.merge(first_name: 'Test', last_name: 'McTest'))
```

Comment:
Adding `first_name` and `last_name` so that Credorax can see the populated `j5` and `j13` fields in our requests. Their sandbox returns a success whether or not these fields are present, but Credorax informs us that downstream processors can reject payments if they are absent.